src/server_config.h

```diff
@@ -249,6 +249,46 @@ int nc_server_config_add_truststore_cert(const struct ly_ctx *ctx, const char *c
 int nc_server_config_del_truststore_cert(const char *cert_bag_name,
         const char *cert_name, struct lyd_node **config);
 
+/**
+ * @brief Gets the hostkey algorithms supported by the server.
```

Comment:
Would be useful to always mention the module of the data, for all the functions.